client/test/testMetadata.ts

```diff
@@ -15,6 +15,7 @@ export default function testMetadata(
     },
     ourPlayerIndex: 0,
     spectating: true,
+    replay: false,
```

Comment:
Why this new metadata? If it's an in-game replay (spectating = false) shouldn't be an issue. If it's a post-game replay, we should be turning on spectating and recalculating the whole game from the beginning (which is a current bug too).

Reply:
I added it, as I added the _replay_ field to _GameMetadata_.
This, in turn, was done to solve #1446 - cardReducer must check if it's a replay to calculate _handsSeeingCardForFirstTime_. (check _spectating_ is not enough)

Reply:
I think there's a disconnect here - on a post-game replay (not in-game) you should always be spectating. Maybe a better name for the flag is "playing" and the value is the opposite of what it is now.

Reply:
Well, currently on a post-game replay spectating===false. (That's what creates that bug...)

Reply:
This is now fixed